### DISCUSSION TITLE
My code seems fine, but why can't I retrieve data from the context?

### DISCUSSION THREAD

**Discussion by Lubette** [rrousselGit/provider#902] [Q&A] [answered]
![241220_16h37m51s_screenshot](https://github.com/user-attachments/assets/46ca6824-b550-4ff0-ae65-fc2daa01aa02)
![241220_16h43m28s_screenshot](https://github.com/user-attachments/assets/8b0352af-ba15-49d0-96d4-41f676bd06b9)
![241220_16h46m01s_screenshot](https://github.com/user-attachments/assets/c8006ae4-3f97-445e-a571-85e658f3019d)

My code is more or less like this, but I don't know why there's an error. I hope someone who is good at this can help me figure it out

**Comment by rrousselGit** [comment, marked as answer]
Make sure to specify the type argument on your provider.
